functional/FunctionalTest.java

```diff
@@ -435,6 +435,26 @@ public static void removeObject_test() throws Exception {
   }
 
   /**
+   * Test: removeObject(final String bucketName, final Iterable&lt;String&gt; objectNames).
+   */
+  public static void removeObject_test2() throws Exception {
+    System.out.println("Test: removeObject(final String bucketName, final Iterable<String> objectNames)");
+
+    String[] filenames = new String[4];
+    for (int i = 0; i < 3; i++) {
+      String filename = createFile(1 * MB);
+      client.putObject(bucketName, filename, filename);
+      Files.delete(Paths.get(filename));
+      filenames[i] = filename;
+    }
+    filenames[3] = "nonexistent-object";
+
+    for (Result<?> r : client.removeObject(bucketName, Arrays.asList(filenames))) {
```

Comment:
The API requires `Iterable<String>`.  `Arrays,asList()` creates an `iterable` object.